moped-editor/src/queries/project.js

```diff
@@ -47,6 +47,7 @@ export const SUMMARY_QUERY = gql`
         feature_signals(where: { is_deleted: { _eq: false } }) {
           signal_id
           knack_id
+          id
```

Comment:
this is unrelated to this issue, but I forgot to query for the feature id in the summary query when I fixed the unique key error https://github.com/cityofaustin/atd-moped/pull/1143

Reply:
thanks for catching that!